commands/public/avatar.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+module.exports = {
+    name: 'avatar',
+
+    description: 'displays a picture of a mentioned users avatar',
+    execute: async function (client, message){
+
+        let member = message.mentions.members.first();
```

Comment:
Change it to
```suggestion
        let member = message.mentions.members.first() || message.guild.members.get(args[0]);
```
This means that you can now also display someone their avatar by the user their Discord ID